test/inference/repgradelbo_distributionsad.jl

```diff
@@ -14,7 +14,7 @@ end
     @testset "$(modelname) $(objname) $(realtype) $(adbackname)" for realtype in
                                                                      [Float64, Float32],
         (modelname, modelconstr) in Dict(:Normal => normal_meanfield),
-        n_montecarlo in [1, 10],
+        n_montecarlo in [10],
```

Comment:
A comment explaining why this is in a list would be good. Is there a plan to expand it to multiple elements, like it was before?

Reply:
I think we actually won't need this in the for loop. Let me demote it into a local variable.